org.hlwd.biblemultithelife.metainfo.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.hlwd.biblemultithelife</id>
```

Comment:
`hlwd.org` does NOT exist.

yet you claim it's controlled by upstream. What is it?

Reply:
I named my namespaces like that, it was also in org.hlwd.sonofman according to Google guidelines.  
Do you think I have to by this DNS ?

Reply:
The documentation is clear. It has to be based on a domain controlled by the **upstream** developers of the application. You even acknowledged it. (the checkbox is not a CAPTCHA)

We are not Google.

Reply:
If there is a problem about it, I can buy the DNS :)